From: Andreas Pehrson <apehrson@mozilla.com>
Date: Fri, 20 Aug 2021 13:52:00 +0200
Subject: Bug 1654112 - Don't check the calling thread in
 webrtc::AudioReceiveStream::GetSources. r=ng

source_tracker_ is thread safe with its own internal mutex, so this call is safe
as long as the caller has a guarantee for the lifetime of the
AudioReceiveStream. This is similar to webrtc::VideoReceiveStream.

Upliftable.

Differential Revision: https://phabricator.services.mozilla.com/D123226
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/c186df8a088e46285a15e40149182daa34cc6805
---
 audio/audio_receive_stream.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/audio/audio_receive_stream.cc b/audio/audio_receive_stream.cc
index 1e8cff5441..32a8e1c172 100644
--- a/audio/audio_receive_stream.cc
+++ b/audio/audio_receive_stream.cc
@@ -368,7 +368,6 @@ int AudioReceiveStreamImpl::GetBaseMinimumPlayoutDelayMs() const {
 }
 
 std::vector<RtpSource> AudioReceiveStreamImpl::GetSources() const {
-  RTC_DCHECK_RUN_ON(&worker_thread_checker_);
   return source_tracker_.GetSources();
 }
 
